stamina-testkit/src/test/scala/stamina/testkit/ScalatestTestGenerationSpec.scala

```diff
@@ -59,7 +59,7 @@ class ScalatestTestGenerationSpec extends StaminaTestKitSpec {
 
     "the sample is only for version 2 and up" should {
       val spec = new StaminaTestKit with WordSpecLike {
-        val persisters = Persisters(ItemPersister[V2]("item1"))
+        val persisters = Persisters(ItemPersister("item1"))
         "TestDomainSerialization" should {
           persisters.generateTestsFor(
             sample(item1).from[V2]
```

Comment:
Should we move this sample and test from this scenario to `a sample is available for both version 1 and (the migrated event) version 2`? It would look something like that:

```scala
    "a sample is available for both version 1 and (the migrated event) version 2" should {
      val spec = new StaminaTestKit with WordSpecLike {
        val persisters = Persisters(ItemPersister("item1"))
        "TestDomainSerialization" should {
          persisters.generateTestsFor(
            sample(item1),
            sample("from-V2", item1).from[V2]
          )
        }
      }

      "generate tests for missing sample data for all versions" in {
        spec.testNames should contain("TestDomainSerialization should deserialize the stored serialized form of Item version 1")
        spec.testNames should contain("TestDomainSerialization should deserialize the stored serialized form of Item version 2")
        spec.testNames should contain("TestDomainSerialization should deserialize the stored serialized form of Item from-V2 version 2")
      }

      "not generate test for version 1 for sample from V2" in {
         spec.testNames shouldNot contain("TestDomainSerialization should deserialize the stored serialized form of Item from-V2 version 1")
      }

      "execute test for missing version 2 from-V2 sample data" in {
         val myRep = execSpec(spec)
         val Some(res) = myRep.findResultEvent("TestDomainSerialization should deserialize the stored serialized form of Item version 2")
         res should be(anInstanceOf[TestFailed])
      }
  }
```